org.thecrossingchurch.CustomJobs/Jobs/HubspotIntegrationMatching.cs

```diff
@@ -0,0 +1,718 @@
+            Debug.WriteLine( string.Format( "Comparing {0} Top-Level Domains.", tlds.Length ) );
+
+            // Get HS API Token
+            string attrKey = dataMap.GetString("AttributeKey");
+            string attrValue = GlobalAttributesCache.Get().GetValue(attrKey);
+            key = Encryption.DecryptString(attrValue);
+
+            // Don't see this being used for anything...
+            var current_id = 0;
+
+            // Setup Excel
+            ExcelPackage excel = new ExcelPackage();
+            ExcelWorksheet worksheet = SetupWorksheet(excel);
+            var row = 2;
+
+            // Setup Context & Services
+            RockContext _context = new RockContext();
```

Comment:
Can we change this to use a using statement so that it is automatically disposed of?